manifests/params.pp

```diff
@@ -25,7 +25,7 @@
   $manage_selinux               = pick($manage_selinux, false)
   $package_ensure               = 'present'
   $module_workdir               = pick($module_workdir,'/tmp')
-  $password_encryption          = versioncmp($version, '14') ? { -1 => 'md5', default => 'scram-sha-256' }
+  $password_encryption          = pick($password_encryption, versioncmp($version, '14') ? { -1 => 'md5', default => 'scram-sha-256' })
```

Comment:
Does that work? `pick()` refrences the variable itself? And line 21-24 follow the same pattern :thinking: 

should it be:
```suggestion
  $password_encryption          = pick($postgresql::globals::password_encryption, versioncmp($version, '14') ? { -1 => 'md5', default => 'scram-sha-256' })
```

Reply:
Yes, I was also surprised. I think it's (ab)using the parser a bit. I wanted to remain consistent.

Reply:
okay. so it's probably coming from globals. interesting. Not sure if that works on purpose or by accident :D

Reply:
"yes"